src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -77,11 +77,28 @@ class Test(avocadoTest):
     def __init__(self, *args, **kwargs):
         """Initialize a Test object."""
         super(Test, self).__init__(*args, **kwargs)
+
+        # if self.timeout is a string any time value may be entered
+        # but the time speciified must be in this order DHMS
+        # examples   10D2H10M10S or 25M or 2H5S
+        if isinstance(self.timeout, str):
+            self.timeout = str(self.timeout).lower()
+            pattern = r""
+            for interval in ("days", "hours", "minutes", "seconds"):
+                pattern += r"(?:(\d+)(?:\s*{0}[{1}]*\s*)){{0,1}}".format(
+                    interval[0], interval[1:])
+            dhms = re.search(pattern, self.timeout).groups()
+            self.timeout = 0
```

Comment:
(pylint-redefined-variable-type) Redefinition of self.timeout type from str to int